### PR TITLE
[Botskills] Update documentation with new schema changes

### PR DESCRIPTION
**Pull request by LCarbonellRod**
### Purpose
*What is the context of this pull request? Why is it being done?*
Since a new manifest schema was released, some documentation became obsolete, so we update it.

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*
* Update the `connect`, `list`, `disconnect`, and `update` documentation.

### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

#### Bots
- [ ] I have validated that new and updated responses use appropriate [Speak](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-text-to-speech?view=azure-bot-service-3.0) and [InputHint](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-add-input-hints?view=azure-bot-service-3.0) properties to ensure a high-quality speech-first experience
- [ ] I have replicated language model changes across the English, French, Italian, German, Spanish, and Chinese `.lu` files and validated that deployment is successful

#### Deployment Scripts
- [ ] I have replicated my changes in the **Virtual Assistant Template** and **Sample** projects
- [ ] I have replicated my changes in the **Skill Template** and **Sample** projects

### CODE REVIEW

**Comment by Batta32** [comment]
@LCarbonellRod is it missing the documentation from [_docs](https://github.com/microsoft/botframework-solutions/tree/master/docs/_docs), right? (which is the one from github docs)

File: tools/botskills/docs/commands/connect.md [hunk 1]
```diff
@@ -35,33 +35,33 @@ For Skills that require other Authentication connection configuration please fol
...
 | --lgOutFolder [path]          | (OPTIONAL) Path for the LuisGen output (defaults to a 'service' folder inside your Virtual Assistant's folder)                                                                      |
-| --skillsFile [path]           | (OPTIONAL) Path to your Virtual Assistant Skills configuration file (defaults to the `skills.json` inside your Virtual Assistant's folder)                                           |
 | --resourceGroup [name]        | (OPTIONAL) Name of your Virtual Assistant's resource group in Azure (defaults to your Virtual Assistant's bot name)                                                                 |
-| --appSettingsFile [path]      | (OPTIONAL) Path to your appsettings file (defaults to `appsettings.json` inside your Virtual Assistant's folder)                                                                    |
+| --appSettingsFile [path]      | (OPTIONAL) Path to your appsettings file (defaults to `appsettings.json` inside your Virtual Assistant's folder. Also, **the skills are stored here**)                                                                    |
```
Comment (Batta32) [inline tools/botskills/docs/commands/connect.md]:
```suggestion
| --appSettingsFile [path]      | (OPTIONAL) Path to your appsettings file where the skills are stored (defaults to `appsettings.json` inside your Virtual Assistant's folder)                                                                    |
```

File: tools/botskills/docs/commands/connect.md [hunk 2]
```diff
@@ -35,33 +35,33 @@ For Skills that require other Authentication connection configuration please fol
...
-| -l, --localManifest \<path>   | Path to local Skill Manifest file                                                                                                                                                   |
-| -r, --remoteManifest \<url>   | URL to remote Skill Manifest                                                                                                                                                        |
+| -l, --localManifest [path]   | Path to local Skill Manifest file                                                                                                                                                   |
+| -r, --remoteManifest [url]   | URL to remote Skill Manifest                                                                                         |
+| -e, --endpointName [name]  | (OPTIONAL) Name of the endpoint to connect to your assistant (case sensitive)(defaults to using the first endpoint) (Intended V2 manifests)                                   |
```
Comment (Batta32) [inline tools/botskills/docs/commands/connect.md]:
```suggestion
| -e, --endpointName [name]  | (OPTIONAL) Name of the endpoint to connect to your assistant which is case sensitive (defaults to using the first endpoint, used for V2 manifest)                                   |
```

File: tools/botskills/docs/commands/connect.md [hunk 3]
```diff
@@ -35,33 +35,33 @@ For Skills that require other Authentication connection configuration please fol
...
-| --appSettingsFile [path]      | (OPTIONAL) Path to your appsettings file (defaults to `appsettings.json` inside your Virtual Assistant's folder)                                                                    |
+| --appSettingsFile [path]      | (OPTIONAL) Path to your appsettings file (defaults to `appsettings.json` inside your Virtual Assistant's folder. Also, **the skills are stored here**)                                                                    |
 | --cognitiveModelsFile [path]  | (OPTIONAL) Path to your Cognitive Models file (defaults to `cognitivemodels.json` inside your Virtual Assistant's folder)                                                           |
 | --verbose                     | (OPTIONAL) Output detailed information about the processing of the tool                                                                                                             |
 | -h, --help                    | Output usage information                                                                                                                                                            |
 
 An example on how to use it with a local Skill manifest file:
```
Comment (Batta32) [inline tools/botskills/docs/commands/connect.md]:
```suggestion
An example on how to use it with a local manifest file:
```

File: tools/botskills/docs/commands/connect.md [hunk 4]
```diff
@@ -35,33 +35,33 @@ For Skills that require other Authentication connection configuration please fol
...
 
 ```bash
-botskills connect --localManifest "./skills/customSkill/customSkillManifest.json" --skillsFile "./skills.json" --cs --verbose
+botskills connect --localManifest "./skills/customSkill/customSkillManifest.json" --appSettingsFile "./appsettings.json" --cs
```
Comment (Batta32) [inline tools/botskills/docs/commands/connect.md]:
```suggestion
botskills connect --localManifest "./skills/customSkill/customSkillManifest.json"  --cs
```

File: tools/botskills/docs/commands/connect.md [hunk 5]
```diff
@@ -35,33 +35,33 @@ For Skills that require other Authentication connection configuration please fol
...
 ```
 
-> **Note:** The paths to both the Skill Manifest and the Virtual Assistant Skills configuration file can be relative or absolute paths equally, and should be explicitly a `.json` file.
+> **Note:** The paths to both the Skill Manifest and the appsettings file, where the Skills are stored, can be relative or absolute paths equally, and should be explicitly a `.json` file.
```
Comment (Batta32) [inline tools/botskills/docs/commands/connect.md]:
```suggestion
> **Note:** The paths to the Skill Manifest can be relative or absolute paths equally, and should be explicitly a `.json` file.
```

File: tools/botskills/docs/commands/connect.md [hunk 6]
```diff
@@ -35,33 +35,33 @@ For Skills that require other Authentication connection configuration please fol
...
 
 An example on how to use it with a remote Skill manifest:
 
 ```bash
-botskills connect --remoteManifest "http://<YOUR_SKILL_MANIFEST>.azurewebsites.net/api/skill/manifest" --skillsFile "./skills.json" --cs --verbose
+botskills connect --remoteManifest "http://<YOUR_SKILL_MANIFEST>.azurewebsites.net/api/skill/manifest" --appSettingsFile "./appsettings.json" --cs
```
Comment (Batta32) [inline tools/botskills/docs/commands/connect.md]:
```suggestion
botskills connect --remoteManifest "http://<YOUR_SKILL_MANIFEST>.azurewebsites.net/api/skill/manifest" --cs
```

File: tools/botskills/docs/commands/disconnect.md [hunk 7]
```diff
@@ -18,19 +18,19 @@ botskills disconnect [options]
 | --cs                          | Determine your Virtual Assistant project structure to be a csharp-like structure                                                                                            |
 | --ts                          | Determine your Virtual Assistant project structure to be a TypeScript-like structure                                                                                        |
 | --noRefresh                   | (OPTIONAL) Determine whether the model of your skills connected are not going to be refreshed (by default they are refreshed)                                               |
+| --appSettingsFile [path]      | (OPTIONAL) Path to your appsettings file (defaults to `appsettings.json` inside your Virtual Assistant's folder. Also, **the skills are stored here**)
```
Comment (Batta32) [inline tools/botskills/docs/commands/disconnect.md]:
```suggestion
| --appSettingsFile [path]      | (OPTIONAL) Path to your appsettings file where the skills are stored (defaults to `appsettings.json` inside your Virtual Assistant's folder)                                                                    |
```

File: tools/botskills/docs/commands/disconnect.md [hunk 8]
```diff
@@ -18,19 +18,19 @@ botskills disconnect [options]
...
+| --lgOutFolder [path]          | (OPTIONAL) Path for the LuisGen output (defaults to a 'service' folder inside your Virtual Assistant's folder)                                                              |                            |
 | --cognitiveModelsFile [path]  | (OPTIONAL) Path to your Cognitive Models file (defaults to `cognitivemodels.json` inside your Virtual Assistant's folder)                                                   |
 | --verbose                     | (OPTIONAL) Output detailed information about the processing of the tool                                                                                                     |
 | -h, --help                    | Output usage information                                                                                                                                                    |
 
 An example on how to use it:
 
 ```bash
-botskills disconnect --skillId "customSkill" --cs
+botskills disconnect --skillId "customSkill" --appSettingsFile "./appsettings.json" --cs
```
Comment (Batta32) [inline tools/botskills/docs/commands/disconnect.md]:
```suggestion
botskills disconnect --skillId "customSkill" --cs
```

File: tools/botskills/docs/commands/disconnect.md [hunk 9]
```diff
@@ -18,19 +18,19 @@ botskills disconnect [options]
...
 ```
 
-> **Note:** The path to the Virtual Assistant Skills configuration file can be relative or absolute path, and should be explicitly a `.json` file.
\ No newline at end of file
+> **Note:** The path to the appsettings file, where the Skills are stored, can be relative or absolute path, and should be explicitly a `.json` file.
```
Comment (Batta32) [inline tools/botskills/docs/commands/disconnect.md]: This can be deleted, this is optional

File: tools/botskills/docs/commands/list.md [hunk 10]
```diff
@@ -13,6 +13,6 @@ botskills list [options]
 
 | Option                   | Description                                                             |
 |--------------------------|-------------------------------------------------------------------------|
-| -f, --skillsFile [path]  | (OPTIONAL) Path to the Virtual Assistant Skills configuration file      |
+| --appSettingsFile [path]      | (OPTIONAL) Path to your appsettings file (defaults to `appsettings.json` inside your Virtual Assistant's folder. Also, **the skills are stored here**)
```
Comment (Batta32) [inline tools/botskills/docs/commands/list.md]:
```suggestion
| --appSettingsFile [path]      | (OPTIONAL) Path to your appsettings file where the skills are stored (defaults to `appsettings.json` inside your Virtual Assistant's folder)                                                                    |
```

File: tools/botskills/docs/commands/update.md [hunk 11]
```diff
@@ -14,33 +14,33 @@ botskills update [options]
...
 | --lgOutFolder [path]          | (OPTIONAL) Path for the LuisGen output (defaults to a 'service' folder inside your Virtual Assistant's folder)                                                              |
-| --skillsFile [path]           | (OPTIONAL) Path to your Virtual Assistant Skills configuration file (defaults to the `skills.json` inside your Virtual Assistant's folder)                                  |
 | --resourceGroup [name]        | (OPTIONAL) Name of your Virtual Assistant's resource group in Azure (defaults to your Virtual Assistant's bot name)                                                         |
-| --appSettingsFile [path]      | (OPTIONAL) Path to your appsettings file (defaults to `appsettings.json` inside your Virtual Assistant's folder)                                                            |
+| --appSettingsFile [path]      | (OPTIONAL) Path to your appsettings file (defaults to `appsettings.json` inside your Virtual Assistant's folder. Also, **the skills are stored here**)                         |
```
Comment (Batta32) [inline tools/botskills/docs/commands/update.md]:
```suggestion
| --appSettingsFile [path]      | (OPTIONAL) Path to your appsettings file where the skills are stored (defaults to `appsettings.json` inside your Virtual Assistant's folder)                                                                    |
```

File: tools/botskills/docs/commands/update.md [hunk 12]
```diff
@@ -14,33 +14,33 @@ botskills update [options]
...
 |-------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| -l, --localManifest \<path>   | Path to local Skill Manifest file                                                                                                                                           |
-| -r, --remoteManifest \<url>   | URL to remote Skill Manifest                                                                                                                                                |
+| -l, --localManifest [path]   | Path to local Skill Manifest file                                                                                                                                           |
+| -r, --remoteManifest [url]   | URL to remote Skill Manifest                                                                                |
+| -e, --endpointName [name]  | (OPTIONAL) Name of the endpoint to connect to your assistant (case sensitive)(defaults to using the first endpoint) (Intended V2 manifests)                                   |
```
Comment (Batta32) [inline tools/botskills/docs/commands/update.md]:
```suggestion
| -e, --endpointName [name]  | (OPTIONAL) Name of the endpoint to connect to your assistant which is case sensitive (defaults to using the first endpoint, used for V2 manifest)                                   |
```

File: tools/botskills/docs/commands/update.md [hunk 13]
```diff
@@ -14,33 +14,33 @@ botskills update [options]
...
+| --appSettingsFile [path]      | (OPTIONAL) Path to your appsettings file (defaults to `appsettings.json` inside your Virtual Assistant's folder. Also, **the skills are stored here**)                         |
 | --cognitiveModelsFile [path]  | (OPTIONAL) Path to your Cognitive Models file (defaults to `cognitivemodels.json` inside your Virtual Assistant's folder)                                                   |
 | --verbose                     | (OPTIONAL) Output detailed information about the processing of the tool                                                                                                     |
 | -h, --help                    | Output usage information                                                                                                                                                    |
 
 An example on how to use it with a local Skill manifest file:
 
 ```bash
-botskills update --localManifest "./skills/customSkill/customSkillManifest.json" --skillsFile "./skills.json" --cs --verbose
+botskills update --localManifest "./skills/customSkill/customSkillManifest.json" --appSettingsFile "./appsettings.json" --cs
```
Comment (Batta32) [inline tools/botskills/docs/commands/update.md]:
```suggestion
botskills update --localManifest "./skills/customSkill/customSkillManifest.json" --cs
```

File: tools/botskills/docs/commands/update.md [hunk 14]
```diff
@@ -14,33 +14,33 @@ botskills update [options]
...
 ```
 
-> **Note:** The paths to both the Skill Manifest and the Virtual Assistant Skills configuration file can be relative or absolute paths equally, and should be explicitly a `.json` file.
+> **Note:** The paths to both the Skill Manifest and the appsettings file, where the Skills are stored, can be relative or absolute paths equally, and should be explicitly a `.json` file.
 
 An example on how to use it with a remote Skill manifest:
 
 ```bash
-botskills update --remoteManifest "http://<YOUR_SKILL_MANIFEST>.azurewebsites.net/api/skill/manifest" --skillsFile "./skills.json" --cs --verbose
+botskills update --remoteManifest "http://<YOUR_SKILL_MANIFEST>.azurewebsites.net/api/skill/manifest" --appSettingsFile "./appsettings.json" --cs
```
Comment (Batta32) [inline tools/botskills/docs/commands/update.md]:
```suggestion
botskills update --remoteManifest "http://<YOUR_SKILL_MANIFEST>.azurewebsites.net/api/skill/manifest" --cs
```

File: tools/botskills/docs/commands/update.md [hunk 15]
```diff
@@ -14,33 +14,33 @@ botskills update [options]
...
 
 An example on how to use it with a local Skill manifest file:
 
 ```bash
-botskills update --localManifest "./skills/customSkill/customSkillManifest.json" --skillsFile "./skills.json" --cs --verbose
+botskills update --localManifest "./skills/customSkill/customSkillManifest.json" --appSettingsFile "./appsettings.json" --cs
 ```
 
-> **Note:** The paths to both the Skill Manifest and the Virtual Assistant Skills configuration file can be relative or absolute paths equally, and should be explicitly a `.json` file.
+> **Note:** The paths to both the Skill Manifest and the appsettings file, where the Skills are stored, can be relative or absolute paths equally, and should be explicitly a `.json` file.
```
Comment (Batta32) [inline tools/botskills/docs/commands/update.md]:
```suggestion
> **Note:** The paths to the Skill Manifest can be relative or absolute paths equally, and should be explicitly a `.json` file.
```

**Comment by LCarbonellRod** [comment]
> @LCarbonellRod is it missing the documentation from [_docs](https://github.com/microsoft/botframework-solutions/tree/master/docs/_docs), right? (which is the one from github docs)

No @Batta32 , in this case the `botskills'` documentation is inside the botskills' folder. Do you want that I move the `botskills'` documentation to the folder that you mentioned [above](https://github.com/microsoft/botframework-solutions/tree/master/docs/_docs)?

**Comment by Batta32** [comment]
No, as you can see in docs folder, there is an explanation which the process of the Virtual Assistant connection to a Skill using botskills which is outdated with the new schema.

**Comment by Batta32** [comment]
Should this PR point to master branch?

**Comment by LCarbonellRod** [comment]
> Should this PR point to master branch?

Yes, since it would solve the [Issue#2602](https://github.com/microsoft/botframework-solutions/issues/2602)

**Pull request closed without merging**